dali/kernels/imgproc/color_manipulation/brightness_contrast_gpu.h

```diff
@@ -78,6 +77,17 @@ TensorListShape<ndims - 1> FlattenChannels(const TensorListShape<ndims> &shape)
 }
 
 
+template <int ndim>
+ivec<ndim - 2> pitch_flatten_channels(const TensorShape<ndim> &shape) {
+  ivec<ndim - 2> ret;
+  int stride = shape[ndim - 1];  // channels
+  for (int i = ndim - 2; i > 0; i--) {
```

Comment:
```suggestion
  for (int i = ndim - 2; i >= 0; i--) {
```
shouldn't it be that?